charts/telegraf-operator/templates/issuer.yml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.certManager.certificate.enable -}}
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
+metadata:
+  name: selfsigned
```

Comment:
Should this be a `telegraf-operator-issuer`?

Reply:
done